README.md

```diff
@@ -28,30 +28,30 @@ yarn add @shopify/polaris
 
 ### Usage
 
-1.  Include the CSS in your HTML:
+1.  Import the CSS directly into your project if your asset packager supports it:
+
+```js
+import '@shopify/polaris/styles.css';
+```
+
+Otherwise include the CSS in your HTML. We suggest copying the styles file into your own project, but you may also use it directly:
```

Comment:
> We suggest copying the styles file into your own project

Do we want to recommend this because we don't want to be tied to `unpkg`? I think if we clarify why we suggest this, it will help people understand why it would be worth copying vs linking in their HTML.